torch_xla/experimental/custom_kernel.py

```diff
@@ -931,24 +928,24 @@ def validate_ragged_paged_attention_inputs(
     raise ValueError(f"{num_q_heads=} must be divisible by {num_kv_heads=}")
 
   # Must check below on runtime!
-  if num_seqs > max_num_seqs:
-    raise ValueError(f"{num_seqs=} must be less or equal to {max_num_seqs=}")
-  max_kv_len = torch.max(kv_lens)
-  min_pages_per_seq = ceil_div(max_kv_len, page_size)
-  if pages_per_seq < min_pages_per_seq:
-    raise ValueError(
-        f"{pages_per_seq=} must be greater or equal to"
-        f" {min_pages_per_seq=} given {max_kv_len=} and {page_size=}.")
-  if cu_q_lens[num_seqs] > max_num_batched_tokens:
-    raise ValueError(
-        f"Total q tokens {cu_q_lens[num_seqs]} must be less or equal to"
-        f" {max_num_batched_tokens=}.")
-  for i in range(num_seqs):
-    q_len = cu_q_lens[i + 1] - cu_q_lens[i]
-    kv_len = kv_lens[i]
-    if q_len > kv_len:
-      raise ValueError(
-          f"{q_len=} must be less or equal to {kv_len=} at sequence {i}.")
+  # if num_seqs > max_num_seqs:
```

Comment:
debug cruft

Reply:
Thanks, @robertgshaw2-redhat , will remove it in the following PR. We have to get it merged today to get a night torch/xla version that can be used for vLLM.